src/components v2/Molecules/Forms/FormControl/FormControl.vue

```diff
@@ -68,9 +68,13 @@ export default Vue.extend({
       default: '',
     },
     size: {
-      type: String as PropType<string>,
+      type: String,
       default: 'md',
     },
+    disabled: {
+      type: Boolean,
```

Comment:
If removal of `PropType` above was intentional, would it better to have kept it and added here for consistency across the library?